domains/batman.is-cool.dev.json

```diff
@@ -0,0 +1,14 @@
+{
+    "domain": "is-cool.dev",
+    "subdomain": "batman",
+
+    "owner": {
+        "email": "Mxpnnngtn9@gmail.com"
+    },
+
+    "record": {
+        "A": ["astro-boilerplate-flame.vercel.app"]
```

Comment:
Invalid A record. A records are IPv4's in an array. What you have appears to a CNAME. Note we do not support Vercel services as per our readme.md guidelines.